src/main/java/edu/cornell/kfs/fp/document/validation/impl/CuDisbursementVoucherDocumentPreRules.java

```diff
@@ -64,4 +96,151 @@ protected boolean checkWireTransferTabState(DisbursementVoucherDocument dvDocume
         return tabStatesOK;
     }
 
+    /**
+     * This method notifies the user if the Check Stub Text will break into more lines than those
+     * allowed in the FastTrack file format and optionally allows the user to go back and fix the issue.
+     * If the user chooses to fix the issue, it will return false, otherwise true.
+     *
+     * @param dvDocument submitted disbursement voucher document
+     * @return Returns true if the user wishes to correct the Check Stub Text, false otherwise.
```

Comment:
After further examination, I noticed that this line's comment is the opposite of what the paragraph above says about the return values. I would strongly recommend fixing these comments for consistency about when the method returns true and when it returns false.